app/controllers/resources_controller.rb

```diff
@@ -6,78 +6,20 @@ class ResourcesController < ApplicationController
   before_action :authorize_request
 
   # POST /objects
-  # rubocop:disable Metrics/AbcSize
-  # rubocop:disable Metrics/MethodLength
   def create
     authorize! :resource
-    begin
-      response_cocina_obj = Dor::Services::Client.objects.register(params: cocina_model)
-    rescue Dor::Services::Client::UnauthorizedResponse => e
-      return render build_error('Error registering object with dor-services-app', e, '500', true)
-    rescue Dor::Services::Client::UnexpectedResponse => e
-      return render build_error('Error registering object with dor-services-app', e, '502')
-    rescue Dor::Services::Client::ConnectionFailed => e
-      return render build_error('Unable to reach dor-services-app', e, '504', true)
-    end
-
-    # Doing this here rather than IngestJob so that have accurate timestamp for milestone.
-    begin
-      workflow_client.create_workflow_by_name(response_cocina_obj.externalIdentifier, 'registrationWF', version: 1)
-    rescue Dor::WorkflowException => e
-      return render build_error('Error creating registrationWF with workflow-service', e, '502')
-    end
-
-    result = BackgroundJobResult.create
-    IngestJob.perform_later(druid: response_cocina_obj.externalIdentifier,
-                            filesets: params[:structural].to_unsafe_h.fetch(:contains, []),
+    result = BackgroundJobResult.create(output: {})
```

Comment:
I wonder if it's worth tossing a simple test of this new interaction into the deleted request spec below.

Reply:
Hmmm ... the request spec wasn't deleted. The diff was just to large to display. Is the spec OK or are you suggesting testing something additional?

Reply:
@justinlittman mea culpa. Yep, that's totally sufficient.